apps/client/src/client/packet.clj

```diff
@@ -36,6 +36,14 @@
              (filter #(= "Ready" (:type %))) first :lastTransitionTime)]
     last-transition-time))
 
+(defn status->external-ips
+  [status]
+  (let [ips
+        (->> status :resources :MachineStatus :addresses
+             (filter #(= "ExternalIP" (:type %))))]
+    (println "ips:" ips)
```

Comment:
Nice pipe! Do we need the `(println)` on 44?

Reply:
Thank you, I think we need some half-pipes in clojure tho :smirk::skateboard:

Removed in caa8bc4